assets/stylesheets/index.scss

```diff
@@ -13,10 +13,12 @@
 @import 'components/app-header';
 @import 'components/badge';
 @import 'components/card';
+@import 'components/data-list';
 @import 'components/icon';
 
 // Layouts
```

Comment:
I think the layout style should come before component style https://nimblehq.co/compass/development/code-conventions/css/#stylesheets-structure

Reply:
fixed in [7f4867e](https://github.com/carryall/go-google-scraper-challenge/pull/73/commits/7f4867e2deebd90680f1f00e8c5b60a89a6484b0)